src/main/java/org/alliancegenome/curation_api/model/entities/ConditionRelation.java

```diff
@@ -0,0 +1,31 @@
+package org.alliancegenome.curation_api.model.entities;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonView;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import org.alliancegenome.curation_api.base.entity.BaseGeneratedAndUniqueIdEntity;
+import org.alliancegenome.curation_api.view.View;
+import org.eclipse.microprofile.openapi.annotations.media.Schema;
+import org.hibernate.envers.Audited;
+
+import java.util.List;
+
+@Audited
+@Data
+@EqualsAndHashCode(onlyExplicitlyIncluded = true)
+@Schema(name = "Disease_Relation", description = "Condition class ")
+public class ConditionRelation extends BaseGeneratedAndUniqueIdEntity  {
+
+    @JsonView({View.FieldsOnly.class})
+    @JsonProperty("condition_relation_type")
+    private String relationType;
+
+    @JsonView({View.FieldsAndLists.class})
+    private List<Condition> conditions;
```

Comment:
@cmpich are these relations in the DB? if so they need @ManytoOne or @OnetoMany annotations on them.